packages/e2e-tests/persistent-seeder-deployment/build-push-release.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -euf -o pipefail
+BASEDIR=$(dirname "$0")
+BACKUP_DOCKERIGNORE="$BASEDIR"/../../../bk.dockerignore
+GLOBAL_DOCKERIGNORE="$BASEDIR"/../../../.dockerignore
+
+switchDockerIgnore () {
+  # necessary for as long as circleci wont support docker 19.03, which adds multiple .dockerignore files support
+  # https://ideas.circleci.com/ideas/CCI-I-1265
+  # https://stackoverflow.com/a/57774684/6569950
+  echo "Switching dockerignore"
+  cp "$BASEDIR"/../../../.dockerignore "$BACKUP_DOCKERIGNORE"
+  cp persistent-seeder.dockerfile.dockerignore "$GLOBAL_DOCKERIGNORE"
+}
+
+restoreDockerIgnore() { #resets, if necessary, the .dockerignore file to it's original state
+  if test -f "$BACKUP_DOCKERIGNORE"; then
+    echo "Restoring dockerignore backup"
+    mv "$BACKUP_DOCKERIGNORE" "$GLOBAL_DOCKERIGNORE"
+  fi
+}
+
+
+# MAIN SCRIPT
+switchDockerIgnore
```

Comment:
I don't think we need this switching dance. I think it's ok for `simple-hub` and this to have the same docker context. The context size will be bigger, but the slowdown is likely not significant enough for added code complexity.

My proposal is here: https://github.com/statechannels/monorepo/pull/1632